### DISCUSSION TITLE
Unable to upload files to my release of my project

### DISCUSSION THREAD

**Discussion by drweb86** [community/community#112850] [Repositories]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Hello

I have issues with my butil project here.
https://github.com/drweb86/butil/releases/tag/2024.03.16

When I try to upload a file to release page, it hangs for 10 minutes and then writes abstract error that something went wrong and that i should try other file.

I tried already for 3 hours and ended up with sharing release files at OneDrive via sharing link, but being unable to upload release files that's major issue for me.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by drweb86** [comment]
Bug is still active.

![image](https://github.com/community/community/assets/4712173/6b02ef6e-a6fa-4948-96ec-026cbaa49552)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @drweb86 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by drweb86** [comment]
@queenofcorgis , Thanks.

**Comment by drweb86** [comment]
Issue is still there, although several days passed.

**Comment by drweb86** [comment]
I setup upload to my NAS. But I dislike this solution. Upload of files still doesnt work.

**Comment by drweb86** [comment]
11 days passed and issue is still not resolved.

**Comment by drweb86** [comment]
18 days passed and issue is still not resolved.

**Comment by drweb86** [comment]
Guys please fix it to Friday please.

**Comment by drweb86** [comment]
Issue is still here.

**Reply by Doomster14** [reply]
Virus? Or false positive for virus? Did you check the file you tried to upload? Did you try uploading another, totally innocent file, just for testing? Can you upload the file anywhere and share a link?

**Comment by drweb86** [comment]
> Virus? Or false positive for virus? Did you check the file you tried to upload? Did you try uploading another, totally innocent file, just for testing? Can you upload the file anywhere and share a link?

Basically release are all files listed here
https://drweb86.synology.me:88/?release=2024.04.05

I uploaded x64 setup to virus total, here're the results
https://www.virustotal.com/gui/file/9256e33e7ea748080b5ddac6bb8c4d3fb13e743a07d93dabe3b13c1f7ccc64b8

**Comment by drweb86** [comment]
Seems like here is a queue.... A month get passed. A month anniversary to this topic.
<img width="151" alt="image" src="https://github.com/community/community/assets/4712173/129b5f0f-bc4c-401a-a664-d826b081e640">

**Comment by onepiecefreak3** [comment]
I encountered this issue now as well. Exactly the same as described. Same behaviour, same tests conducted. I would really like to release my applications.

If the error could be more telling about what went wrong.
I really hate those vague error messages, I mean you know what went wrong, don't you? You don't just randomly not accept files and sweep it under the rug. Just tell us what went wrong.

**Reply by drweb86** [reply]
Hi.

My current workaround is the next

I created web-site on my personal NAS with following content
https://github.com/drweb86/butil/tree/master/sources/web

During release I update latest.json and put files.

Hope it will be fixed one day.

**Reply by onepiecefreak3** [reply]
This solution is not viable for me. I don't have a NAS and don't want to provide storage or availability.
That's what the release tab on github is for.

But thanks for the suggestion anyway.

**Comment by onepiecefreak3** [comment]
I researched this issue further and it seems to be directly related to my internet connection.

Uploading release binaries through my home network is basically impossible with an unmeasuringly low upload rate, but going via mobile network, which is not associated with my home network, the upload works fine.

Since the issue only persists since I wrote here and the last time something on my home network changed (router, cables, etc) was over 3 months ago, I have to assume that connection to me or my ISP are maybe blocked or throttled for certain actions?

I can pull and push code normally, but maybe the throttle is unnoticable there, as the delta information is just too small to notice. I'm really at a loss for words, especially since this ONLY pertains to github currently. I can upload the exact same binary/release package on other platforms and cloud storage providers without any throttling, so it definitely has to do with the communication to github in some way.

@drweb86 Is it possible for you to confirm, that this might also affect you in the same way? Try using a different network to upload the releas builds to github.

**Reply by drweb86** [reply]
I tried using VPN, but it failed. On my phone i have same internet. But I'm happy that you managed to solve the issue.

**Reply by onepiecefreak3** [reply]
I wouldn't call it a solution. It's a somewhat easy-to-do workaround for me.
But the issue itself should still be resolved.

**Reply by revoconner** [reply]
It didn't work for me at all. Sometimes it works without doing anything, sometimes no matter what you do.

**Comment by revoconner** [comment]
<img width="1745" alt="image" src="https://github.com/community/community/assets/88772846/200a1a79-4cfa-4eae-b3e6-b765440e92ad">
I tried with different networks, VPN enabled, disabled, different browsers. I keep getting this, will it ever be solved?

**Comment by drweb86** [comment]
Seems like queue of issues is incredibly huge. So we missed celebration of 68 days in the queue!

<img width="244" alt="image" src="https://github.com/community/community/assets/4712173/88f72083-f028-49c9-bec1-5a26f01415d5">

**Comment by drweb86** [comment]
Issue seems to be resolved.

**Comment by coco-is-magik** [comment]
I am still having this issue. It looks like there is an interrupt happening when I try to upload something, similar to the earlier screenshot.

**Comment by mateokladaric** [comment]
still an issue

**Comment by NiordFresh** [comment]
bruh, still having this issue...

**Reply by revoconner** [reply]
A workaround I found is turning off the strict security policy of browser for github solved this. 

On safari its called content blockers and on firefox its strict security policy. No idea about chromium.

If these are turned on, the network log shows error in connection to some of network requests on github. 

It's also applicable when you try to run a github codespace. With these enabled it'll just keep reconnecting after a few seconds of loading UI. 

As for why that is, I haven't got a clue.

**Reply by NiordFresh** [reply]
<img width="679" height="601" alt="obraz" src="https://github.com/user-attachments/assets/dc31dfc0-bd57-46e3-b554-143ac2bd0725" />


hmm... I had Ultra Protection set here initially, after disabling it and setting the DNS in the OS to CloudFlare (1.1.1.1), the problem disappeared...
